### PR TITLE
Use libtwirl for install and stuff

### PR DESCRIPTION
**Pull request by mugman174**
I meant to merge this instead of the libtwirl thing but ehh

### CODE REVIEW

**Review by caughtquick — APPROVED** [review]
Looks good

**Review by caughtquick — DISMISSED** [review]
Looks good thanks for fixing the uninstall I started

File: twrl [hunk 1]
```diff
@@ -13,39 +15,40 @@ except Exception:
 	print(f"Provide an operation (e.g. {sys.argv[0]} search)")
```
Comment (thewarsawpakt) [inline twrl:15]: Use `twirl.errors.IncorrectArgumentFormatError` here, instead of a print.

**Comment by mugman174** [comment]
@caughtquick @thewarsawpakt fixed

**Review by thewarsawpakt — APPROVED** [review]
Approved.

**Review by caughtquick — APPROVED** [review]
Reapproved merging now

**Comment by mugman174** [comment]
:D thx

**Pull request merged**